src/calendar/index.tsx

```diff
@@ -26,8 +26,6 @@ type MarkedDatesType = {
 };
 
 export interface CalendarProps extends CalendarHeaderProps, DayProps {
-  /** Specify theme properties to override specific styles for calendar parts */
-  theme?: Theme;
```

Comment:
Is it safe to remove all these props?

Reply:
They are header props, but I'll put them back and we'll sort these props issues in another occasion

Reply:
Done